app/controllers/users_controller.rb

```diff
@@ -0,0 +1,39 @@
+class UsersController < ApplicationController
+  include UsersHelper
+  def index
+    @users = User.all
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
+    # if @user.save
+    #   redirect_to new_user_path
+    # else
+    #   render :new
+    # end
```

Comment:
- [ ] Comments: It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history.